### ISSUE TITLE
Running workspace has a start button. Stopping a workspace shows two buttons

### ISSUE THREAD

**Issue by bpmct** [coder/coder#13560]
Marking as `s1` to prioritize a fix before we release next.

<img width="1781" alt="Screenshot 2024-06-12 at 2 40 31 PM" src="https://github.com/coder/coder/assets/22407953/9a901a87-f115-4868-80a5-297fa0393074">

**Comment by aslilac** [comment]
Closed by #13572

**Comment by bpmct** [comment]
I don't think this is entirely fixed, when I stop a workspace I see two buttons still.

cc @stirby

**Comment by bpmct** [comment]
I mean _technically_ it is a different issue, but I didn't see an issue for this and it feels like we need to tackle the root cause

**Comment by stirby** [comment]
@bpmct are you still seeing the issue?

**Comment by BrunoQuaresma** [comment]
Not seeing this anymore

**Comment by stirby** [comment]
A customer is encountering this **daily** right now on `v2.12.3`. @BrunoQuaresma I think something slipped through the cracks when we merged the start and update buttons.

**Comment by stirby** [comment]
Resolved.
